test/quadratic.jl

```diff
@@ -49,7 +49,7 @@ xlo, xhi = itp3[.2], itp3[xmax+.7]
 @test_approx_eq xhi A[end]
 
 # Check continuity
-xs = [0:.1:length(A)+1]
+xs = [0:.1:length(A)+1;]
```

Comment:
Ah. I had completely missed out on that discussion :P

I must admit that I don't like the syntax very much, although I realize I'm likely to be overridden by consensus here (and it might grow on me with time). Would I be considered terribly conservative if I perferred `[i-1 for i in 1:.1:length(x)]` (or `[(0:.1:length(x)-1)...]`) over `[0:.1:length(x)-1;]`? I just think the semicolon is a little too magical...